document-store/build.gradle.kts

```diff
@@ -22,7 +22,6 @@ dependencies {
 
   testImplementation(libs.org.junit.jupiter.junit.jupiter)
   testImplementation(libs.org.mockito.mockito.core)
-  testImplementation(libs.org.mockito.mockito.inline)
```

Comment:
Newer versions of mockito have this built in. The old version was not compatible with newer java.